app/scripts/files/templates/file.html

```diff
@@ -61,15 +61,17 @@ <h3 class="panel-title pull-left" data-translate>File Properties</h3>
               </tr>
               <tr>
                 <th scope="row" data-translate>Archive</th>
-                <td>
-                  {{ ::fc.file.archive.archive_id || '--' }}
-                  <span data-ng-if="fc.file.archive">
-                    (<a data-ui-sref="search.files({ 'filters': {{
+                <td data-ng-if="fc.file.archive">
+                  {{ ::fc.file.archive.submitter_id || '--' }}
+                  - rev {{ ::fc.file.archive.revision || '--'}}
+                  (<a data-ui-sref="search.files({ 'filters': {{
                       [{ field: 'files.archive.archive_id', value: fc.file.archive.archive_id }] | makeFilter }}
                       })">
                     {{ fc.archiveCount }} files
                   </a>)
```

Comment:
moved the ng-if to the td level if there are 0 files it'll not show this and show https://github.com/NCI-GDC/portal-ui/pull/1014/files#diff-3acfaf0591334ca8b64227bfc57a39d5R73